lib/web_ui/dev/browser_lock.yaml

```diff
@@ -21,9 +21,19 @@ firefox:
 edge:
   launcher_version: '1.2.0.0'
 ios-safari:
+  # Make sure this version is the same version supported by LUCI macOS bots.
+  # XCode on these bots will be updated once a year, do not forget to update
+  # `heightOfHeader` during this time.
   majorVersion: 13
   minorVersion: 0
   device: 'iPhone 11'
+  # `xcrun simctl` command is used to take screenshots. It takes the screenshot
+  # of the entire simulator. Therefore we need to all the parts other than the
+  # browsers' content. Do not forget to update these values during simulator
+  # version upgrade.
+  # `heightOfHeader` is the number of pixels taken by the phone's header menu
+  # and the browsers address bar.
+  heightOfHeader: 282

```

Comment:
I'll skip this suggestion. I think it's unnecessary complexity for now. I couldn't find any simulator which supports this case. May be there would be in the future but considering 2 things, it is better to not to spend that effort for now:
- It's very hard to change simulators and Xcode, we don't know if we will have a new one (with new simulators which have gaps on left and right side) any time in the next quarters
- Meanwhile `simctl` getting an update where it can provide better APIs for media is more likely.